test/unit/deletion_test.rb

```diff
@@ -3,42 +3,52 @@
 class DeletionTest < ActiveSupport::TestCase
   self.use_transactional_fixtures = false # Disabled to test after_commit
 
+  should belong_to :user
+
   setup do
     @user = create(:user)
     Pusher.new(@user, gem_file).process
     @version = Version.last
   end
 
-  test "must be indexed" do
+  should "must be indexed" do
```

Comment:
since you're touching this line anyway, maybe change that to `should "be indexed"`